vdb-bench-assembly/plugins/vdb-bench-core/TranslatorSelectionService.js

```diff
@@ -42,7 +42,30 @@
             "teiid" : "plugins/vdb-bench-core/content/img/Teiid_dv_logos_70x40.png",
             "ws" : "plugins/vdb-bench-core/content/img/WebService_dv_logos_70x40.png"
         };
-        
+
+        var raTranslators = ["simpledb", 
+                             "accumulo", 
+                             "solr",
+                             "cassandra",
+                             "file",
+                             "google-spreadsheet",
+                             "infinispan-cache-dsl",
+                             "infinispan-cache",
+                             "jpa2",
+                             "ldap",
+                             "loopback",
+                             "excel",
+                             "mongodb",
+                             "map-cache",
+                             "odata",
+                             "odata4",
+                             "olap",
+                             "salesforce",
+                             "salesforce-34",
+                             "ws",
+                             "sap-gateway",
+                             "sap-nw-gateway"];
+
```

Comment:
Wondering if this list should be fetched from komodo. Feels like this is business stuff being put into a client that shouldn't really need to know. Also, makes future maintenance more complicated since not everyone will know this list is included in the client.
WDYT